Scripts/AzureAppConfig/validate_aac_secrets.py

```
@@ -0,0 +1,318 @@
+            "ERROR", entry_location(path, index, key, label),
+                    f"Value looks like a credential ({description}). "
+                    f"Move it to Key Vault and reference it instead."))
+                break
+
+    # An endpoint-only key carrying StackExchange.Redis configuration syntax.
+    # EndPoints.Add() cannot parse it, so this breaks the connection as well as
+    # being the shape a pasted credential arrives in.
+    if is_endpoint_only_key(key) and "," in value:
+        findings.append(Finding(
+            "WARN", entry_location(path, index, key, label),
+            f"Value contains ',' but this key is assigned to "
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)